frontend/src/view/audio-annotator/components/spectro-render.component.tsx

```diff
@@ -338,6 +344,9 @@ export const SpectroRenderComponent: React.FC<Props> = ({audioPlayer,}) => {
         canvas.height
       ));
 
+    // Color spectro image
+    colorSpectro(canvas, spectroContext.currentColormap.colormap, spectroContext.currentColormap.invertColors);
```

Comment:
To simplify the function call, it could directly take "spectroContext.currentColormap" as second parameter ?